### DISCUSSION TITLE
input formats to create PMTiles

### DISCUSSION THREAD

**Discussion by wiesehahn** [protomaps/PMTiles#92] [Q&A] [answered]
Do I always have to convert my data to MBTiles first to generate PMTiles?

**Comment by bdon** [comment, marked as answer]
Right now, yes... what tool are you using to create MBTiles? the long term goal is to have tools output PMTiles directly.

**Reply by wiesehahn** [reply]
I have to admit I never did so far. Just trying to use gdal_translate to convert COG to MBTiles. So gdal driver with write capabilities for PMTiles would be nice I guess.

**Reply by bdon** [reply]
agreed that GDAL read/write support is a good long term goal, I don't have any immediate plans to build it right now unless a company wants to sponsor it. So I'd recommend GDAL MBTiles output -> `pmtiles convert`
